src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.tsx

```diff
@@ -159,7 +159,7 @@ const AvailablePipelinesDrawer: React.FC<AvailablePipelinesDrawerProps> = ({
           />
         </Container>
         <Text font={{ variation: FontVariation.BODY2 }} color={Color.GREY_500}>
-          {getString('cf.featureFlags.flagPipeline.drawerDescription')}
+          <String stringID="cf.featureFlags.flagPipeline.drawerDescription" useRichText />
```

Comment:
Unfortunately a `<p>` tag doesn't allow things like lists inside it (it'll render, but it won't be happy about it). You should probably tell the `Text` component to be a `<div>` instead
```suggestion
        <Text font={{ variation: FontVariation.BODY2 }} color={Color.GREY_500} tag="div">
          <String stringID="cf.featureFlags.flagPipeline.drawerDescription" useRichText />
```

Reply:
huh okay didn't know that, updated